app/assets/stylesheets/contents_table.css

```diff
@@ -0,0 +1,19 @@
+.table-of-contents {
+  width: 100%;
+  background: #eee;
+  padding: 2em;
```

Comment:
Can the top and bottom padding be a little less? Looks a bit stark at the moment `padding: 0.1em 2em` looks good.